.github/workflows/static-code-analysis.yml

```diff
@@ -0,0 +1,13 @@
+name: Static code analysis
+
+on: [push]
```

Comment:
`on: [push, pull_request]`

Reply:
Did think about this as well, but can't really get my head around in what situation you would want to add `pull_request`. Doesn't `push` cover each situation already?

Reply:
"push" is just a push to this repo. 
We do want to run this when someone else push to their fork and then make a pull request, right?

Reply:
Ie, if you made this code change on your fork and then made the PR, you would not see this: 

![Screenshot 2019-12-05 at 12 13 20](https://user-images.githubusercontent.com/1275206/70230461-a91c8180-1758-11ea-88ab-49ff2e9f9440.png)

Reply:
I use `pull_request` only! :D
https://github.com/odolbeau/phone-number-bundle/blob/master/.github/workflows/static.yml#L1

Reply:
Added `pull_request` 👍